miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/RunService.java

```diff
@@ -1,8 +1,87 @@
 package uk.ac.bbsrc.tgac.miso.service.impl;
 
+import java.io.IOException;
+import java.util.Collection;
+import java.util.Map;
+
+import com.eaglegenomics.simlims.core.Note;
+import com.eaglegenomics.simlims.core.User;
+
 import uk.ac.bbsrc.tgac.miso.core.data.Run;
+import uk.ac.bbsrc.tgac.miso.core.data.RunQC;
+import uk.ac.bbsrc.tgac.miso.core.data.type.QcType;
 import uk.ac.bbsrc.tgac.miso.core.util.PaginatedDataSource;
+import uk.ac.bbsrc.tgac.miso.service.security.AuthorizationException;
 
 public interface RunService extends PaginatedDataSource<Run> {
 
+  Collection<Run> list() throws IOException;
+
+  Collection<Run> listWithLimit(long limit) throws IOException;
+
+  int count() throws IOException;
+
+  Map<String, Integer> getRunColumnSizes() throws IOException;
+
+  Collection<Run> listBySearch(String query) throws IOException;
```

Comment:
Do we need these (and the sample versions)? Can we use the pagination interface to do that? We might need sequencer and container, but I think the rest are covered.